summary_workflows/quantification/specification/Q_benchmark_specification.md

```diff
@@ -190,15 +190,16 @@ The following tables list the metric names, value types and units, and a descrip
 
 | Metric_id | Type | Unit | Description |
 | :--- | :--- | :--- | :----------------- |
-| `Correlation_coefficient_Pearson_10nt` | `float` | N/A | Correlation between poly(A) site quantification by benchmarked tool and ground truth dataset calculated for 10 nt distance threshold (window) |
-| `Correlation_coefficient_Pearson_50nt` | `float` | N/A | Correlation between poly(A) site quantification by benchmarked tool and ground truth dataset calculated for 50 nt distance threshold (window) |
-| `Correlation_coefficient_Pearson_100nt` | `float` | N/A | Correlation between poly(A) site quantification by benchmarked tool and ground truth dataset calculated for 100 nt distance threshold (window) |
+| `Correlation_coefficient_Pearson_with_unmatched_GT_10nt` | `float` | N/A | Correlation between poly(A) site quantification by benchmarked tool and ground truth dataset calculated for 10 nt distance threshold (window) |
+| `Correlation_coefficient_Pearson_with_unmatched_GT_50nt` | `float` | N/A | Correlation between poly(A) site quantification by benchmarked tool and ground truth dataset calculated for 50 nt distance threshold (window) |
+| `Correlation_coefficient_Pearson_with_unmatched_GT_100nt` | `float` | N/A | Correlation between poly(A) site quantification by benchmarked tool and ground truth dataset calculated for 100 nt distance threshold (window) |
 | `Expression_non-matched-PAS_10nt` | `float` | N/A | Total expression of non-matched sites quantified by benchmarked tool for 10 nt distance threshold (window) |
 | `Expression_non-matched-PAS_50nt` | `float` | N/A | Total expression of non-matched sites quantified by benchmarked tool for 50 nt distance threshold (window) |
 | `Expression_non-matched-PAS_100nt` | `float` | N/A | Total expression of non-matched sites quantified by benchmarked tool for 100 nt distance threshold (window) |
-| `Correlation_coefficient_Pearson_relative_100nt` | `float` | N/A | Correlation between relative PAS usage calculated from RNAseq-based PAS quantification and orthogonal 3'end seq data for 100 nt distance threshold (window) |
+| `Correlation_coefficient_Pearson_relative_with_unmatched_GT_and_PD_100nt` | `float` | N/A | Correlation between relative PAS usage calculated from RNAseq-based PAS quantification and orthogonal 3'end seq data for 100 nt distance threshold (window) |
 
```

Comment:
Are we possibly missing the description of without unmatched in this table?

Reply:
Mh in this table in the specification only metrics are described that appear in the plots. The note below mentions that all three versions are computed and available in the `assessment_dataset.json` and in the `consolidated_result.json`.
My intend was to keep the computations as they were but include the additional ones in the json files. Once we settle on the metric to plot and to use, we can adjust it here in the specification. 
Do you agree?

Reply:
Ah i see! That makes sense! I thought it was a list of the metrics we have now hehe if that's the case then I was thinking maybe we could add somewhere a consolidated list of the possible metric calculation options and their descriptions, but I guess that should be a separate PR :D